### DISCUSSION TITLE
Deep source bot contribution

### DISCUSSION THREAD

**Discussion by realamirhe** [chintan9/plyr-react#675] [Q&A] [answered]
I wonder why deep source bot contribution to the project is only one commit? 

do we have any dashboard? If so could you please add me there?

- [Dashboard overview](https://deepsource.io/docs/dashboard/dashboard-overview)
- [Managing members in your team](https://deepsource.io/docs/access-control/team-members)

**Comment by chintan9** [comment, marked as answer]
Let me look into docs
